### PR TITLE
Moving `correlationId` and `merchant_account_id` up one level

### PR DESCRIPTION
**Pull request by jonathajones**
Thank you for your contribution to Braintree. 

### Summary of changes

-  This PR fixes the nesting of two keys in the Native Checkout module. Previously, the `getBaseParameters` function returned a dictionary where two keys were mapped at an incorrect depth. This PR fixes that, and correctly moves `correlation_id` and `merchant_account_id` up one level in the parameters dictionary.

### Checklist

- [X] Added a changelog entry

- @jonathajones

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Braintree iOS SDK Release Notes
 
+## Unreleased
+* Fixed the nesting of two keys in the `getBaseParameters` dictionary in the BraintreePayPalNativecheckout module
```
Comment (scannillo) [inline CHANGELOG.md:4]:
```suggestion
* BraintreePayPalNativeCheckout (BETA)
  * Fix `merchant_account_id` and `correlation_id` keys to be nested at the top level of the internal create order request
```

Comment (scannillo) [reply CHANGELOG.md:4]: Changes look good to me! Just a small comment on CHANGELOG styling to match our previous entries.

**Review by jaxdesmarais — APPROVED** [review]
Just a small change to the changelog header otherwise looks great! Thanks for making this update. 🚀

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (BETA)
+  * Fixed the nesting of two keys in the `getBaseParameters` dictionary in the BraintreePayPalNativecheckout module
```
Comment (scannillo) [inline CHANGELOG.md:5]:
```suggestion
  * Fix `merchant_account_id` and `correlation_id` keys to be nested at the top level of the internal create order request
```

I think this suggestion got lost in the original thread here

**Review by scannillo — APPROVED** [review]
Small CHANGELOG comment to include explicit key names, otherwise LGTM! Thanks for making this update 👏

**Pull request merged**